docs/index.md

```diff
@@ -6,24 +6,22 @@ Privacy is having the agency to decide when you reveal personal information. It
 
 Loki provides the means to transact and communicate privately and anonymously, using the decentralised nature of blockchain technology to create new methods of interacting with the internet.
 
-Loki is a private cryptocurrency based on Monero. Over the course of the coming months, the Loki project aims to offer an incenvised full node layer with a secondary p2p network that offers a private communications layer based on the Tor protocol.
+Loki is a private cryptocurrency based on Monero. Over the course of the coming months, the Loki project aims to offer an incenvised full node layer with a secondary peer-to-peer network that offers a private communications layer based on the Tor protocol.
```

Comment:
perhaps refer to this https://github.com/loki-project/loki/pull/298

Reply:
Changed to reflect changes from loki-project/loki#298